app/controllers/chores_controller.rb

```diff
@@ -0,0 +1,56 @@
+class ChoresController < ApplicationController
+  def create
+    title = params[:title]
+    description = params[:description]
+    due_date = params[:due_date]
+    assignee_email = params[:assignee_email]
+    hideout_id = params[:hideout_id]
+
+    if title.nil? || hideout_id.nil?
+      render status: 400
+    else
+      if assignee_email.nil?
+        assignee_id = nil
+      else
+        assignee_id = User.find_by(email: assignee_email).id
+      end
+      chore = Chore.create(title: title, description: description, due_date: due_date, assignee_id: assignee_id, hideout_id: hideout_id)
+      chore.save
+      render status: :created, :json => {title: title, description: description, due_date: due_date, assignee_email: assignee_email, hideout_id: hideout_id}
+    end
+  end
+
+  def update
+    params.require(:id)
+    id = params[:id]
+
+    # TODO: add middleware for checking if user exists and chores exists 
```

Comment:
I don't know if we would need to check if user exists. I'm thinking we'll need to make sure the request comes with a token that signals that the user has access to the hideout and so they can make the change required. @Ram-Raghav-S  what do you think?

Reply:
I agree, we don't need to make sure the user exists. I think we can just get a token identifying the user that sent the request and check whether the user is in the hideout - how does that sound?

Reply:
Yeah that works